examples/pylint/C0121_singleton_comparison.py

```diff
@@ -3,7 +3,7 @@ def is_true():
     @rtype: bool
     """
     temp = 5
-    if temp is None:
+    if temp == None:
```

Comment:
I believe this check's purpose is to make sure you compare against singletons with `is` instead of `==`